overlord/ifacestate/hotplug.go

```diff
@@ -178,6 +178,23 @@ func suggestedSlotName(devinfo *hotplug.HotplugDeviceInfo, fallbackName string)
 	return shortestName
 }
 
+// updateDevice creates tasks to disconnect slots of given device, update the slot in the repository, then connect it back.
+func updateDevice(st *state.State, ifaceName, hotplugKey string, newAttrs map[string]interface{}) *state.TaskSet {
+	hotplugDisconnect := st.NewTask("hotplug-disconnect", fmt.Sprintf("Disable connections of device %q", hotplugKey))
```

Comment:
Mmh,  this doesn't mention even the interface,  we should probably try to add something along the concept of a human understandable/showable subset of the hotplugKey. Also probably mention the slot names as well whenever possible.

Reply:
Small tweak - https://github.com/snapcore/snapd/pull/6315 - to make it consistent with other hotplug summaries. Will revisit once we agree on details.